src/libAtomVM/md5.c

```diff
@@ -0,0 +1,225 @@
+// https://github.com/Zunawe/md5-c/
```

Comment:
This is not compliant with SPDX and REUSE.
When we import 3rd party sources we need to track the original license.
Also when we vendor something I really suggest to mention git revision or tag we took this from, so it is easier to track upstream.
Last, but not least, I really suggest vendoring crypto stuff from popular projects such as mbedTLS, rather than relying on some random dev on github.